force-app/main/default/lwc/pdftronAttachmentPickerCombobox/pdftronAttachmentPickerCombobox.js

```diff
@@ -1,145 +1,182 @@
-import { LightningElement, track, wire, api } from 'lwc';
-import { CurrentPageReference } from 'lightning/navigation';
-import { fireEvent,registerListener, unregisterAllListeners } from 'c/pubsub';
-import getAttachments from "@salesforce/apex/PDFTron_ContentVersionController.getAttachments";
-import getBase64FromCv from "@salesforce/apex/PDFTron_ContentVersionController.getBase64FromCv";
-import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+import { LightningElement, track, wire, api } from 'lwc'
+import { CurrentPageReference } from 'lightning/navigation'
+import { fireEvent, registerListener, unregisterAllListeners } from 'c/pubsub'
+import { ShowToastEvent } from 'lightning/platformShowToastEvent'
+import getAttachments from '@salesforce/apex/PDFTron_ContentVersionController.getExistingAttachments'
+import getBase64FromCv from '@salesforce/apex/PDFTron_ContentVersionController.getBase64FromCv'
+import apexSearch from '@salesforce/apex/PDFTron_ContentVersionController.search'
+import getFileDataFromId from '@salesforce/apex/PDFTron_ContentVersionController.getFileDataFromId'
 
 export default class PdftronAttachmentPickerCombobox extends LightningElement {
-    error;
-
-    @track value = '';
-    @track picklistOptions = [];
-    @track isSaving = false;
-    @track loadFinished = false;
-    documentsRetrieved = false;
-    @api recordId;
-    @wire(CurrentPageReference) pageRef;
-
-    renderedCallback() {
-        if(!this.documentsRetrieved) {
-            getAttachments({recordId: this.recordId})
-            .then((data) => {
-                console.log('data', data);
-                
-                data.forEach((attachmentRecord) => {
-                    console.log(JSON.stringify(attachmentRecord));
-                    var name = attachmentRecord.cv.Title + "." + attachmentRecord.cv.FileExtension;
-                    const option = {
-                        label: name,
-                        value: attachmentRecord.cv.Id
-                    };
-                    console.log('option', option);
-                    this.picklistOptions = [ ...this.picklistOptions, option ];
-                });
-                this.error = undefined;
-                this.loadFinished = true; 
-                this.documentsRetrieved = true;
-            })
-            .catch((error) => {
-                console.error(error)
-                this.showNotification('Error', error, 'error');
-                this.error = error;
-            }); 
-        }
-    }
+  error
```

Comment:
```suggestion
  error;
```

Curious why the semi-colon was removed in the diff lol

In-fact, I notice a lot of the lines in this file don't end with semi-colons 😅